SplunkAWSCloudWatchStreamingMetricsProcessor/lambda_function.py

```diff
@@ -50,16 +50,52 @@ def transform_json_metric_event(metrics):
 	for metric_string in metrics:
 		metric = ast.literal_eval(metric_string)
 		metric_event = copy.deepcopy(metric)
+		
+		
 		# reformat metric_dimensions
 		extracted_dims = {
```

Comment:
I don't see `extracted_dims` being used again. Any reason `metric_dimensions` are not added to the `metric_event`